resources/cluster-essentials/files/subscriptions.eventing.kyma-project.io.crd.yaml

```diff
@@ -0,0 +1,186 @@
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: SubscriptionSpec defines the desired state of Subscription
+            properties:
+              filter:
+                description: BebFilters defines the list of BEB filters
+                properties:
+                  dialect:
+                    type: string
+                  filters:
+                    items:
+                      description: BebFilter defines the BEB filter element as a combination
+                        of two CE filter elements
+                      properties:
+                        eventSource:
+                          description: Filter defines the CE filter element
+                          properties:
+                            property:
+                              type: string
+                            type:
+                              type: string
+                            value:
+                              type: string
+                          required:
+                          - property
+                          - value
+                          type: object
+                        eventType:
+                          description: Filter defines the CE filter element
+                          properties:
+                            property:
+                              type: string
+                            type:
+                              type: string
+                            value:
+                              type: string
+                          required:
+                          - property
+                          - value
+                          type: object
+                      required:
+                      - eventSource
+                      - eventType
+                      type: object
+                    type: array
+                required:
+                - filters
+                type: object
+              id:
+                description: ID is the unique identifier of Subscription, read-only.
+                type: string
+              protocol:
+                type: string
+              protocolsettings:
+                description: ProtocolSettings defines the CE protocol setting specification
+                  implementation
+                properties:
+                  contentMode:
+                    type: string
+                  exemptHandshake:
+                    type: boolean
+                  qos:
+                    type: string
+                  webhookAuth:
+                    description: Webhook defines the Webhook called by an active subscription
+                      in BEB
```

Comment:
Do you think this is required?

Reply:
It is a valid description of a field. Can you explain your concern in detail?